bedrock/contentful/tests/test_contentful_api.py

```diff
@@ -671,6 +728,41 @@ def test_AssetBlockRenderer(mock_client, mock__get_image_url):
     assert output == expected
 
 
+@pytest.mark.parametrize(
+    "content_type_label",
+    (
+        "embedBlockquote",
+        "embedImage",
+        "somethingElse",
+    ),
+)
+@patch("bedrock.contentful.api.ContentfulPage.client")
+@patch("bedrock.contentful.api._make_image")
+@patch("bedrock.contentful.api._make_blockquote")
+def test_BlockEntryRenderer(
+    mock_make_blockquote,
+    mock_make_image,
+    mock_client,
+    content_type_label,
+):
+    mock_entry = Mock()
+    mock_content_type = Mock()
+    mock_content_type.id = content_type_label
+    mock_entry.sys = {"content_type": mock_content_type}
+    mock_client.entry.return_value = mock_entry
+
+    node = {"data": {"target": {"sys": {"id": mock_entry}}}}
+
+    output = BlockEntryRenderer().render(node)
+
+    if content_type_label == "embedBlockquote":
+        mock_make_blockquote.assert_called_once_with(mock_entry)
+    elif content_type_label == "embedImage":
+        mock_make_image.assert_called_once_with(mock_entry)
+    else:
+        assert output == content_type_label
```

Comment:
This is safe as-is. If the mock is called, it will output the mock instead of content_type_label and the test will fail:
```python
>           assert output == content_type_label
E           AssertionError: assert <MagicMock name='_make_image()' id='4489162272'> == 'embedImage'
```